go/common/enclave.go

```diff
@@ -114,10 +114,11 @@ type Enclave interface {
 
 // BlockSubmissionResponse is the response sent from the enclave back to the node after ingesting a block
 type BlockSubmissionResponse struct {
-	IngestedRollupHeader    *Header                   // The header of the winning rollup contained in the ingested block, if any.
-	ProducedSecretResponses []*ProducedSecretResponse // if L1 block contained secret requests then there may be responses to publish
-	SubscribedLogs          map[rpc.ID][]byte         // The logs produced by the block and all its ancestors for each subscription ID.
-	RejectError             *BlockRejectError         // this is set if block was rejected, contains information about what block to submit next
+	ProducedRollup          *ExtRollup                // If the node is the sequencer, the rollup produced if the ingested L1 block contained a new rollup.
```

Comment:
We send back a produced rollup now, instead of generating it using a separate call.